voter-dapp/src/ActiveRequests.js

```diff
@@ -423,6 +487,7 @@ function ActiveRequests({ votingAccount, votingGateway }) {
                 <TableCell>{formatDate(pendingRequest.time, drizzle.web3)}</TableCell>
                 <TableCell>{statusDetails[index].statusString}</TableCell>
                 <TableCell>{getCurrentVoteCell(index)}</TableCell>
+                <TableCell>{getLastCommittedSalt(index)}</TableCell>
```

Comment:
how does this render on smaller screens? did you test it out on mobile?

Reply:
It does not render well, but the voter dApp in general does not render well on mobile. I think displaying the full salt is overkill so I decided to truncate the salt to the first 6 digits and added a "CopyToClipboard" button for the user's convenience 

![Screen Shot 2020-05-28 at 09 53 16](https://user-images.githubusercontent.com/9457025/83150700-9496c180-a0c9-11ea-8c01-ea3952c6c652.png)